KotlinMemo/src/main/kotlin/com/incubate/kotlinmemo/ConfigurationBean.kt

```diff
@@ -0,0 +1,23 @@
+package com.incubate.kotlinmemo
+
+import com.incubate.kotlinmemo.repository.MemoMemoryRepository
+import com.incubate.kotlinmemo.repository.MemoRepository
+import com.incubate.kotlinmemo.service.MemoService
+import com.incubate.kotlinmemo.service.MemoServiceImpl
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.PropertySource
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories
+import org.springframework.stereotype.Component
+import org.springframework.transaction.annotation.EnableTransactionManagement
+import javax.persistence.EntityManager
+import javax.sql.DataSource
+
+@Configuration
+class ConfigurationBean(private val dataSource: DataSource, private var em: EntityManager) {
+    @Bean
+    fun memoService():MemoService{ return MemoServiceImpl(memoRepository()) }
```

Comment:
많은 정보가 한줄로 들어가 있어서 보기가 편하지는 않은거 같아요.

굳이 필요 없는 표현들을 줄여

```suggestion
    fun memoService() = MemoServiceImpl(memoRepository())
```

정도로 작성을 하시거나 개행을 해서 작성을 하시면 보기 더 편해질거 같아요

Reply:
넵 수정하겠습니당 감사합니다.